CompDMD_Location.m

```diff
@@ -38,8 +39,9 @@
 
 %% output
 parameters = getMultiStarts(parameters, objectiveFunction, optionsPesto);
-optx = reshape(parameters.MS.par(:,1),[N,L]);
-[SPToolboxFolder,~,~]=fileparts(which('CompDMD_Location'))
-filename=sprintf('%s%i%s%i%s','DMDinfo\B_SP_dim',N,'points',L,'.csv');
+optx = reshape(parameters.MS.par(:,1),[N,L-1]);
+optx = [optx,-sum(optx,2)];
```

Comment:
Nice so far. I would just try to have the name a bit more informative here: How about renaming 
optx -> spLocations
?